lib/Notes/NoteCreatePage/NoteCreatePage.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router';
+
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes-core';
+import { Icon } from '@folio/stripes-components';
+
+import NoteForm from '../NoteForm';
+import noteTypesCollectionShape from '../response-shapes';
+
+@stripesConnect
+class NoteCreatePage extends Component {
+  static propTypes = {
+    domain: PropTypes.string.isRequired,
+    entityTypeTranslationKeyMap: PropTypes.objectOf(PropTypes.string),
+    mutator: PropTypes.object,
```

Comment:
Does not this prop is required? Moreover, I would specify it more detailed

```
mutator: PropTypes.shape({
  notes: PropTypes.shape({
      POST: ...
  })
}),
```

Reply:
@VictorSoroka1, you are right, it should be required and described more detailed. Thank you!